### PR TITLE
feat(20.04): add 'copyright' slice to all SDFs

### PR DESCRIPTION
**Pull request by cjdcordeiro**
# Proposed changes

Same as https://github.com/canonical/chisel-releases/pull/228

### Forward porting

https://github.com/canonical/chisel-releases/pull/228
#229 
#230 

## Checklist

* [x] I have read the [contributing guidelines](
https://github.com/canonical/chisel-releases/blob/main/CONTRIBUTING.md)
* [x] I have already submitted the [CLA form](
https://ubuntu.com/legal/contributors/agreement)

### CODE REVIEW

File: slices/liblzma5.yaml [hunk 1]
```diff
@@ -1,7 +1,14 @@
 package: liblzma5
+essential:
+  - liblzma5_copyright
+
```
Comment (rebornplusplus) [inline slices/liblzma5.yaml:4]: nit: it would be nice to have a blank line separating `package` and `essential`.

**Review by rebornplusplus — APPROVED** [review]
Looks good to me. The packages who have copyright files within them, do have the copyright slices added in this PR.

The files who do not have the copyright files for this release are the following:
* libgcc-s1
* libncursesw6
* libpython3.8-stdlib
* libstdc++6

I only left a nit comment about spacing inside the slice definition files. But please feel free to ignore that -- it's a nitpick.

File: slices/liblzma5.yaml (hunk 1 above)
Comment (cjdcordeiro) [reply slices/liblzma5.yaml:5]: done

**Review by letFunny — APPROVED** [review]
Thanks!

**Pull request merged**